src/js/compatibility.js

```diff
@@ -18,23 +18,31 @@
     */
     if (parseInt(playerLibrary.version, 10) >= 8) {
         var jwplayerCompatible = function(query) {
-            var playerInstance = playerLibrary(query);
-            if (!playerInstance.trigger) {
+            var basePlayer = playerLibrary(query);
+            var playerInstance = Object.create(basePlayer);
+            if (!playerInstance.trigger || playerInstance.compatibility) {
                 return playerInstance;
             }
 
             /*
                 We've removed a few methods from the public API, and our events now implement Backbone events.
              */
-            playerInstance.dispatchEvent = playerInstance.trigger;
-            playerInstance.removeEventListener = playerInstance.off;
-            playerInstance.getItem = playerInstance.getPlaylistIndex;
-            playerInstance.getMeta = playerInstance.getItemMeta;
+            playerInstance.dispatchEvent = basePlayer.trigger;
+            playerInstance.removeEventListener = basePlayer.off;
```

Comment:
`playerInstance.removeEventListener` should also be set to  `playerInstance.off.bind(basePlayer)`